src/components/progress-flow/progress-flow-item/progress-flow-item.tsx

```diff
@@ -120,4 +121,9 @@ export class ProgressFlowItem {
 
         return <div class="divider" />;
     }
+    private renderLockIcon() {
+        if (this.isDisabled()) {
+            return <limel-icon name="lock" class="lock-icon" />;
+        }
+    }
```

Comment:
We normally use these kinds of "early return" in the components.
Also added an empty before the block

```suggestion

    private renderLockIcon() {
        if (!this.isDisabled()) {
            return;
        }
        
        return <limel-icon name="lock" class="lock-icon" />;
    }
```